recipes/reg/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "reg" %}
+{% set version = "0.10" %}
+{% set sha256 = "7ac7618d32c6e9921b71614a826fe4ed23887beede99f27e2fc2e3c1c7ad6959" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - repoze.lru
```

Comment:
Is `repoze.lru` needed at build time or only install time? If the latter, please drop this.